src/data_warehouse/cli/transforms.py

```
@@ -3,95 +1,122 @@
+        if store_results:
+            target_path = os.path.join(project_dir, 'target')
+            log_results(target_path, db_url)
+    finally:
+        os.chdir(original_dir)
+
+def log_results(target_path, db_url):
+    """Log DBT test results to the database"""
+    try:
+        # Import here to avoid circular imports
+        script_path = Path(__file__).parent.parent.parent.parent / 'scripts' / 'log_dbt_test_results.py'
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)